src/Peasy/Peasy.Tests/RuleBaseTests.cs

```diff
@@ -444,87 +442,87 @@ public async Task Successor_Does_Not_Invoke_IfInvalidThenExecute_Async()
         public void First_Valid_Rule_In_First_Successor_Chain_Should_Execute()
         {
             var output = string.Empty;
-            var rule = new TrueRule()
-                              .IfValidThenValidate(new TrueRule().IfValidThenExecute(r => output = "pass"), new TrueRule())
-                              .IfValidThenValidate(new TrueRule(), new FalseRule3())
-                              .Validate();
```

Comment:
I think I left the rule declaration there explicitly to help readers understand that .`Validate()` returns a new Rule instance.  Maybe we should leave this as is?  I'm guessing resharper caught this and changes below?

Reply:
I removed it as the return value isn't used is just creating assigning a new instance that has to be managed instead of lost, but I do understand the readability.
I'll revert this change.